Basic-Car-Maintenance/Shared/Onboarding/Views/WelcomeView.swift

```diff
@@ -17,10 +18,10 @@ struct WelcomeView: View {
                     HStack(spacing: 5) {
                         Text("Welcome to")
                         Text("Basic")
-                            .foregroundStyle(Color("basicGreen"))
```

Comment:
note we don't need to use strings anymore because assets automatically generate type safe names like `.basicGreen`

Reply:
You fixed this also, right? I see in commit 'fix colors, capitalization and comments', you changed all `.basicGreen`?